### PR TITLE
Reducing size package

### PR DESCRIPTION
**Pull request by ardguezsoc**
3110f38:
- Due that we keep all the country lists used for the country question we have a lot of unnecessary weight (kb). Each of this files is around 17kb-14kb. Reducing the name of the keys we now have 8-9kb. We could convert maybe this object to an array or another thing to make them even smaller. This would have sense because it is intern logic related with how we handle the data so if we really want to have this possibilities in the library they should be as small as we can. 

e57be95: 

- There was unnecessary import due by default we already have the en language so we dont really need to import this. For each import that we use to have the language it is like 5-10kb more to the package so with this change in the conditional we have reduced the size too.

before
after

### CODE REVIEW

**Review by sofisdev — APPROVED** [review]
LGTM!!

File: src/Fields/Date/index.js [hunk 1]
```diff
@@ -25,13 +24,13 @@ const DatePicker = ({
 }) => {
   const [date, setDate] = React.useState(new Date())
   const pickerRef = React.useRef(null)
-  const mapLanguagues = { de, fr, es, en }
-  const datepickerLanguage =
-    language && mapLanguagues[language.toLowerCase()]
-      ? mapLanguagues[language.toLowerCase()]
-      : mapLanguagues.en
+  const mapLanguagues = { de, fr, es }
```
Comment (ismaelocaramelo) [inline src/Fields/Date/index.js:27]: why this change?

**Review by ismaelocaramelo — COMMENTED** [review]
Just some doubt on this, the rest LGTM

File: src/Fields/Date/index.js (hunk 1 above)
Comment (ardguezsoc) [reply src/Fields/Date/index.js:27]: because we dont need the 'en' it is the default so it would have no sense having the else. if it is not in the maplanguages we will show it in english because the locale is 'en' or we will display it in 'en' because it is another random locale and we dont have the language for it.
